### PR TITLE
Jena Ontology API

### PR DESCRIPTION
**Pull request by sszuev**
new ontapi documentation

### CODE REVIEW

File: source/documentation/ontology/__index.md [hunk 1]
```diff
@@ -205,18 +204,40 @@ doesn't require a number to be the subject of an RDF statement. In
...
-constraints of OWL DL plus some more. The intent for OWL Lite and
-OWL DL is to make the task of reasoning with expressions in that
-subset more tractable. Specifically, OWL DL is intended to be able
-to be processed efficiently by a
+The OWL language is sub-divided into several syntax classes:
+*OWL2 Full*, *OWL2 DL*, *OWL2 RL*, *OWL2 EL*, *OWL2 QL*,
+and also *OWL1 Lite*, *OWL1 DL* and *OWL1 Full*.
+The last three are deprecated now.
+OWL2 EL, OWL2 QL and OWL2 RL do not permit some constructions allowed in OWL2 Full and OWL2 DL.
+Although OWL1 is deprecated, Jena Ontology API is still support them.
```
Comment (kinow) [inline source/documentation/ontology/__index.md:212]: s/is still support them/still supports it (or is still supported)

File: source/documentation/ontology/__index.md [hunk 2]
```diff
@@ -10,6 +10,9 @@ won't go into all of the many details of the API here: you should
 expect to refer to the [Javadoc](/documentation/javadoc/jena/) to
 get full details of the capabilities of the API.
 
+_Please note that this section covers the new Jena ontology API, which has been introduced since Jena 5.1.0.
+The legacy Jena Ontology API documentation see [here](legacy.html)._
```
Comment (kinow) [inline source/documentation/ontology/__index.md:14]: Maybe s/The legacy/For the legacy

Or /API documentation see [here/API documentation can be found [here.../

File: source/documentation/ontology/__index.md (hunk 1 above)
Comment (sszuev) [reply source/documentation/ontology/__index.md]: fixed

File: source/documentation/ontology/__index.md (hunk 2 above)
Comment (sszuev) [reply source/documentation/ontology/__index.md]: fixed

**Comment by afs** [comment]
Preparation for release 5.1.0. 

The target branch changed to "jena-next".

**Pull request merged**